tox.ini

```diff
@@ -0,0 +1,36 @@
+[flake8]
+max-line-length = 88
```

Comment:
Can we set both flake8 and black to 100? I'm not against switching to 88 across the board, but currently the other charms/bundles are using 100.